cfpb/consumerfinance.gov
# Fix Ask CFPB Suggestions for ES7

Ask CFPB is currently suggesting the same term you searched for when it has no other suggestions and there are no results for your search term.


---

## Changes

- Detect when the suggested term is the same as the search term, and return the original state of the search.


## How to test this PR

1. Search a term with no suggestion or results such as "zelle" and confirm that the search results does not suggest searching for "zelle". 
2. Search for a term such as "morgage" and ensure that you get results for "mortgage" and a link to the results for "morgage"


## Screenshots
![image](https://user-images.githubusercontent.com/16838497/100894513-91bac600-348a-11eb-96e3-5e968491fd2c.png)

![image](https://user-images.githubusercontent.com/16838497/100894547-9a130100-348a-11eb-8c0d-ee63dc8d89d4.png)

![image](https://user-images.githubusercontent.com/16838497/100894566-9f704b80-348a-11eb-86c8-5ba489681155.png)


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
